src/net.cpp

```diff
@@ -2355,14 +2355,10 @@ void StartNode(void* parg)
     {
         LogPrintf("Scraper disabled.");
 
-        // If the neural network is specified to be disabled, do not enable housekeeping thread.
-        if (!GetBoolArg("-disablenn", false))
```

Comment:
May also need to update these locations: 

https://github.com/gridcoin-community/Gridcoin-Research/blob/274e822a7266af70445495997350c48a9c514847/src/init.cpp#L524-L530
https://github.com/gridcoin-community/Gridcoin-Research/blob/274e822a7266af70445495997350c48a9c514847/src/init.cpp#L534-L537
https://github.com/gridcoin-community/Gridcoin-Research/blob/274e822a7266af70445495997350c48a9c514847/src/neuralnet/quorum.cpp#L530-L534

Reply:
I looked at those. I think they are ok...

Reply:
It will result in the quorum being deactivated, but the housekeeping thread should still run to ensure that received manifests are cleaned out.